src/styles/index.scss

```
@@ -1,3 +1,148 @@
 body {
   margin: 0;
+  $card-border: #f3f3f3;
+  $color-black: #060b35;
+  $color-gray: #616070;
+  $color-blue: #00acdc;
+  $color-white: #fff;
+  $gap: 4px;
+
+  body,
+  p,
+  a,
+  img,
+  div {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: Roboto, Helvetica, sans-serif;
+  }
+
+  .card {
+    width: 200px;
+    height: 406px;
+    padding: 32px 16px 16px;
+    border-radius: 5px;
+    border: 1px solid $card-border;
+
+    &__image {
+      display: block;
+      width: 160px;
+      height: 134px;
+      margin: 0 auto;
+    }
+
+    &__title {
+      margin-top: 40px;
+      margin-bottom: 0;
+      font-style: normal;
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 500;
+      color: $color-black;
+      overflow: hidden;
+    }
+
+    &__code {
+      margin-top: 4px;
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 14px;
+      color: $color-gray;
+    }
+
+    &__rating {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-top: 16px;
+    }
+
+    &__reviews {
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+    }
+
+    &__cost {
+      display: flex;
+      justify-content: space-between;
+      height: 18px;
+      margin-top: 24px;
+
+    }
+
+    &__price {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $color-gray;
+    }
+
+    &__amount {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $color-black;
+    }
+
+    &__link {
+      display: flex;
+      width: 100%;
+      height: 40px;
+      justify-content: center;
+      align-items: center;
+      margin-top: 16px;
+      font-size: 14px;
+      font-weight: 700;
+      text-transform: uppercase;
+      text-decoration: none;
+      color: $color-white;
+      border-radius: 5px;
+      background-color: $color-blue;
+      border: 1px solid transparent;
+    }
+
+    &__link:hover {
+      color: $color-blue;
+      background-color: $color-white;
+      border-color: $color-blue;
+    }
+  }
+
+  .stars {
+    display: flex;
+
+    &__star {
+      width: 16px;
+      height: 16px;
+      background-image: url(../images/star.svg);
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+
+    &--1 > :nth-child(-n+1) {
+      background-image: url(../images/star-active.svg);
+    }
+
+    &--2 > :nth-child(-n+2) {
+      background-image: url(../images/star-active.svg);
+    }
+
+    &--3 > :nth-child(-n+3) {
+      background-image: url(../images/star-active.svg);
+    }
+
+    &--4 > :nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+    }
+
+    &--5 > :nth-child(-n+5) {
+      background-image: url(../images/star-active.svg);
+    }
```

```suggestion
    &--1 > :nth-child(-n+1) {
      background-image: url(../images/star-active.svg);
    }

    &--2 > :nth-child(-n+2) {
      background-image: url(../images/star-active.svg);
    }

    &--3 > :nth-child(-n+3) {
      background-image: url(../images/star-active.svg);
    }

    &--4 > :nth-child(-n+4) {
      background-image: url(../images/star-active.svg);
    }

    &--5 > :nth-child(-n+5) {
      background-image: url(../images/star-active.svg);
    }
```

You can combine this selectors using comma:

```
.class1,
.class2 {
  your styles here;
]
```